jenkins/disco_app_conf_update.sh

```diff
@@ -14,7 +14,7 @@ if [[ "$1" == "-h" || "$1" == "--help" || "$1" == "" ]] ; then
     exit 1
 fi
 
-source "$(dirname $0)/boto_init.sh"
+source /opt/wgen/bin/use_asiaq_from_rpm.sh
```

Comment:
I think we already do this here https://github.com/amplifylitco/asiaq_astro_config/blob/8ae71b8f8afb7208d35014f4360f26f8599d6c72/discoroot/etc/init.d/disco-jenkins~for_deployenator#L146 .